client/src/components/cityview/realm/SettleRealmComponent.tsx

```diff
@@ -32,14 +33,15 @@ export const SettleRealmComponent = () => {
     let realm = getRealm(new_realm_id);
     let position = getPosition(new_realm_id);
     let entity_id = await create_realm({
+      signer: account,
       owner: import.meta.env.VITE_KATANA_ACCOUNT_1_ADDRESS,
```

Comment:
here owner should then be the burner account address. Normally everything should work even if you change that but we'd have to test it more. maybe leave it like that and create an issue ?

Reply:
@ponderingdemocritus will create new issue for this